doc/reference/networking/can_api.rst

```diff
@@ -0,0 +1,273 @@
+.. _can_interface:
+
+Controller Area Network (CAN)
+#############################
+
+.. contents::
+    :local:
+    :depth: 2
+
+Overview
+********
+
+Controller Area Network is a two-wire serial bus specified by the
+Bosch CAN Specification, Bosch CAN with Flexible Data-Rate specification and the
+ISO 11898-1:2003 standard.
+CAN is mostly known for its application in the automotive domain. However, it
+is also used in home and industrial automation and other products.
+
+A so-called CAN transceiver connects the two bus lines, CAN High (CAN H) and
+CAN Low (CAN L), from the CAN network to the CAN controller.
+The send wire from the controller to the transceiver is called CAN TX, and the
+receive wire is called CAN RX. These wires use the logic levels whereas the bus
+level is interpreted differential between CAN H and CAN L.
+Both lines, CAN H and CAN L, at the same level, is called the recessive state.
+This state is seen as a logical one, which is also the idle state.
+The other state is called the dominant state and is interpreted as a logical
+zero value. To write a dominant bit to the bus, open-drain transistors tight
+CAN H to Vdd and CAN L to ground. The first and last node use a 120 ohm resistor
+between CAN H and CAN L to terminate the bus.
+The termination avoids reflections on the bus and keeps CAN L and CAN H at the
+same level in idle state.
+The dominant state always overrides the recessive state.
+This structure is called a wired-AND.
+
+CAN controllers can only initialize when the bus is in the idle (recessive)
+state for a specific amount of time. Therefore you have to make sure that CAN RX
+is high, at least for a short time. This is also necessary for loopback mode.
+
+CAN uses Identifiers to identify the frame instead of addresses to identify a
```

Comment:
Should the `Identifiers` be written as `identifiers`?

Reply:
Maybe "so-called Identifiers"